index.html

```diff
@@ -132,6 +132,18 @@ <h1 class="header__title">
     <small class="text-muted">PSR-15 Middleware in Minutes</small>
 </h3>
 <hr>
+<div class="toc">
+    <h6 class="toc__headline">On this page</h6>
+    <ul class="toc__list"><li class="toc__entry">
+    <a href="#api" class="toc__link nav-link">API</a>
+</li><li class="toc__entry">
+    <a href="#authentication" class="toc__link nav-link">Authentication</a>
+</li><li class="toc__entry">
+    <a href="#authorization" class="toc__link nav-link">Authorization</a>
+</li><li class="toc__entry">
+    <a href="#session" class="toc__link nav-link">Session</a>
+</li></ul>
+</div>
```

Comment:
I think at some point we should add a linter/style tool for HTML to the project. :smile:

Reply:
And I prefer a GitHub action with PHP. 😃